### ISSUE TITLE
Links

### ISSUE THREAD

**Issue by stevenaproctor** [markwright785/A11YPatterns#4]
**WCAG 2.0 Guidelines**
These are repeated on the form elements and navigation pages but the content of the progressive reveals are identical.

Some of the guidelines have a colon (:) as part of the link. Others have the colon outside the link but separated by a space. It should be [1.1.1 Non-text Content]: All...

These links should open in the same window, or have something to indicate that they are going to open in a new window or tab.

**CSS on GitHub**
These links are repeated on the same pages as the WCAG 2.0 Guidelines.

There is nothing to indicate to a screen reader that these will open in a new tab as the arrow icon is not read out. I would recommend opening them in the same window.

The CSS links would make more sense out of context as:

* View the form elements CSS on GitHub
* View the tables CSS on GitHub
* View the navigation CSS on GitHub
